blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -41,6 +41,8 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Renamed the `timestamp` data type to `timestamp with time zone`.
```

Comment:
Can this be folded into the previous entry regarding the renamed CrateDB data types?